cmd/deptokens/main.go

```diff
@@ -13,22 +13,21 @@ import (
 	"github.com/micromdm/nanodep/tokenpki"
 )
 
-const (
-	defaultCN   = "deptokens"
-	defaultDays = 1
-)
+const defaultCN = "deptokens"
 
 // overridden by -ldflags -X
 var version = "unknown"
 
 func main() {
 	var (
-		flCert     = flag.String("cert", "cert.pem", "path to certificate")
-		flKey      = flag.String("key", "cert.key", "path to key")
-		flPassword = flag.String("password", "", "password to encrypt/decrypt private key with")
-		flTokens   = flag.String("token", "", "path to tokens")
-		flForce    = flag.Bool("f", false, "force overwriting the keypair")
-		flVersion  = flag.Bool("version", false, "print version")
+		flCert         = flag.String("cert", "cert.pem", "path to certificate")
+		flKey          = flag.String("key", "cert.key", "path to key")
+		flCN           = flag.String("cn", defaultCN, "common name to use when creating the certificate")
+		flValidityDays = flag.Int64("validity-days", 0, "validity of the certificate in days")
```

Comment:
default to 1, and some nits

```suggestion
		flDays = flag.Int64("days", 1, "validity of the certificate in days")
```